paper-card.html

```diff
@@ -58,7 +58,7 @@
 
 <dom-module id="paper-card">
   <template>
-    <style>
+    <style include="paper-material">
```

Comment:
Technically, this is great. However, stylistic, I kind of like the first example from the Polymer docs:

```html
<!-- import the module  -->
<link rel="import" href="../shared-styles/shared-styles.html">
<dom-module id="x-foo">
  <template>
    <!-- include the style module by name -->
    <style include="shared-styles"></style>
    <style>:host { display: block; }</style>
    Hi
  </template>
  <script>Polymer({is: 'x-foo'});</script>
</dom-module>
```

I kind of like thinking about these imported stylesheets as distinct modules, rather than a single module that keeps getting extended by external chunks.

This is mostly :bike: :house:. WDYT?